src/journey-maps/angular/services/map/map-poi-service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { Map as MaplibreMap } from 'maplibre-gl';
+
+import { SbbPointsOfInterestOptions } from '../../journey-maps.interfaces';
+
+export const SBB_POIS_LAYER = 'journey-pois';
+const SBB_POIS_LAYER_LIST = [SBB_POIS_LAYER, 'journey-pois-hover', 'journey-pois-selected'];
+
+@Injectable({ providedIn: 'root' })
+export class SbbMapPoisService {
```

Comment:
Just a thought, not a todo: A service for this functionality seems to be overkill, especially since it mutates a parameter. A function or just a method in the SbbFeatureEventListener class might be better.